ariadne/cardano/test/Test/Ariadne/Bip44.hs

```diff
@@ -0,0 +1,153 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+module Test.Ariadne.Bip44
+       ( bip44PathGen
+       , bip44KeyPairGen
+       ) where
+
+import Test.Hspec(Spec, describe)
+import Test.Hspec.QuickCheck (prop)
+import Test.QuickCheck (Property, property, (===), conjoin, (.&&.))
+import Test.QuickCheck.Arbitrary (Arbitrary(..))
+import Test.QuickCheck.Gen (Gen(..))
+
+import Ariadne.Wallet.Cardano.Kernel.Bip44
+       ( Bip44DerivationPath(..)
+       , decodeBip44DerivationPath
+       , encodeBip44DerivationPath)
+import Ariadne.Wallet.Cardano.Kernel.Word31 (Word31, unsafeMkWord31, word31ToWord32)
+import Ariadne.Wallet.Cardano.Kernel.DB.HdWallet
+       ( HdAddressChain (..)
+       , HdAccountIx (..)
+       , HdAddressIx(..)
+       , HdCoinType(..))

```

Comment:
I believe you are not using Ariadne's default [`stylish-haskell` settings](https://github.com/serokell/ariadne/blob/master/.stylish-haskell.yaml). Please reach out to me if you need help configuring them in Sublime.